examples/plot_geodesic_poincare_polydisk.py

```diff
@@ -0,0 +1,71 @@
+"""
+Plot a geodesic on the Poincare polydisk,
+with Poincare Disk visualization.
+"""
+
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import geomstats.backend as gs
+import geomstats.visualization as visualization
+from geomstats.geometry.poincare_polydisk import PoincarePolydisk
+
+two_disks = PoincarePolydisk(n_disks=1, point_type='extrinsic')
```

Comment:
TWO_DISKS

it's a constant variable, which should be all uppercases as per pep8 guidelines